src/main/java/uk/gov/pay/directdebit/mandate/api/ConfirmMandateRequest.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.pay.directdebit.mandate.api;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Map;
+
+public class ConfirmMandateRequest {
+    @JsonProperty("sort_code")
+    private String sortCode;
+    
+    @JsonProperty("account_number")
+    private String accountNumber;
+
+    @JsonProperty("transaction_external_id")
+    private String transactionExternalId;
+
+    public ConfirmMandateRequest(String sortCode, String accountNumber, String transactionExternalId) {
+        this.sortCode = sortCode;
+        this.accountNumber = accountNumber;
```

Comment:
these should be of type `SortCoe` and AccountNumber` now I think?

Reply:
Yes, good catch. I think we used account number and sortcode everywhere except here. I originally thought this would be an object which get serialised (hence the `@jsonproperty`) annotations, but it's actually just a domain object, so no need to have properties as strings